src/bindings/html/dom.js

```diff
@@ -81,19 +81,34 @@ export function translateMutations(view, langs, mutations) {
             targets.add(addedNode);
           }
         }
+        break;
     }
   }
 
-  targets.forEach(
-    target => target.childElementCount ?
-      translateFragment(view, langs, target) :
-      translateElement(view, langs, target));
+  if (targets.size === 0) {
+    return;
+  }
+
+  let elements = [];
+
+  targets.forEach(target => {
```

Comment:
One last stylistic change, please indulge me :) I find the x => { } syntax difficult to read.  Can we make this snippet more lambda-like:

    targets.forEach(target => target.childElementCount ?
      elements.concat(getTranslatables(target) : elements.push(target));

This also has the benefit of being shorter :)